### ISSUE TITLE
$wpdb->delete() can't really be cached

### ISSUE THREAD

**Issue by JDGrimes** [WordPress-Coding-Standards/WordPress-Coding-Standards#180]
Calling `$wpdb->delete()` is flagged as an uncached query. Of course, a delete operation can't really be cached. Same goes for `update` and `insert`. However, I do suppose that in many cases they should perhaps still be accompanied by a cache update.

So perhaps this should be left as it is, but it seems odd to me.

Related #178

**Comment by JDGrimes** [comment]
The PR was merged and released in 0.3.0. I guess this is fixed.

**Comment by samwilson** [comment]
I'm still getting an error with `$wpdb->delete()`. I'm using the latest on the `develop` branch.

**Comment by samwilson** [comment]
Sorry... here's a minimal code example that triggers the error I'm getting:

```php
<?php
/**
 * A demonstration of WPCS failing on $wpdb->delete().
 *
 * @file
 * @package phpcs
 */

$wpdb->delete();
```

**Comment by JDGrimes** [comment]
See b7d5416. `$wpdb->delete()` should probably be accompanied by a `wp_cache_delete()` or `wp_cache_set()` depending on the nature of the cache.

**Comment by JDGrimes** [comment]
So in other words, this was reverted by #400.
